content/en/docs/guides/complex-component-structure-deployment.md

```diff
@@ -0,0 +1,1069 @@
+```
+
+```yaml
+# FluxDeployment
+apiVersion: delivery.ocm.software/v1alpha1
+kind: FluxDeployer
+metadata:
+  name: backend-kustomization
+  namespace: ocm-system
+spec:
+  kustomizationTemplate:
+    prune: true
+    targetNamespace: default
+  sourceRef:
+    apiVersion: delivery.ocm.software/v1alpha1
+    kind: Configuration
+    name: backend-configuration
+    namespace: ocm-system
+```
+
+And that's it.
+
+These components can be found under [podinfo/backend/components](https://github.com/open-component-model/podinfo/tree/f6fd27a94a5cf39784754858bd2a139bd90e0ad9/backend/components).
+
+To apply them, simply run this command from the podinfo root:
+
+```
+kubectl apply -f backend/components
+```
+
+### Frontend
+
+Same for the Frontend
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: Localization
+metadata:
+  name: frontend-localization
+  namespace: ocm-system
+spec:
+  configRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: config
+      referencePath:
+      - name: frontend
+      version: 1.0.0
+  interval: 10m0s
+  sourceRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: manifests
+      referencePath:
+      - name: frontend
+      version: 1.0.0
+```
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: Configuration
+metadata:
+  name: frontend-configuration
+  namespace: ocm-system
+spec:
+  configRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: config
+      referencePath:
+      - name: frontend
+      version: 1.0.0
+  interval: 10m0s
+  sourceRef:
+    apiVersion: delivery.ocm.software/v1alpha1
+    kind: Localization
+    name: frontend-localization
+    namespace: ocm-system
+```
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: FluxDeployer
+metadata:
+  name: frontend-kustomization
+  namespace: ocm-system
+spec:
+  kustomizationTemplate:
+    prune: true
+    targetNamespace: default
+  sourceRef:
+    apiVersion: delivery.ocm.software/v1alpha1
+    kind: Configuration
+    name: frontend-configuration
+    namespace: ocm-system
+```
+
+To apply them, simply run this command from the podinfo root:
+
+```
+kubectl apply -f frontend/components
+```
+
+### Redis
+
+Redis is exactly the same as the above two. Just with different names and pointing to the redis resource. Try creating
+these yourself to see if you understood the structure. If you get stuck, you can always take a peek under
+[podinfo/redis/components](https://github.com/open-component-model/podinfo/tree/f6fd27a94a5cf39784754858bd2a139bd90e0ad9/redis/components).
+
+To apply them, simply run this command from the podinfo root:
+
+```
+kubectl apply -f redis/components
+```
+
+## Understanding the moving parts
+
+How does the whole flow work?
+
+![mpas-flow](/images/mpas-workflow.png)
```

Comment:
again we should avoid any reference to MPAS.

Reply:
Ops I think just named it incorrectly because I was working on maps too. Thanks good spot.